oi/OICodeGen.h

```diff
@@ -51,18 +51,25 @@ struct ParentMember {
 class OICodeGen {
  public:
   struct Config {
+    Config() = default;
+    Config(const Config& other) = delete;
+    Config& operator=(const Config& other) = delete;
+    Config(Config&& other) = default;
+    Config& operator=(Config&& other) = default;
+
     /*
      * Note: don't set default values for the config so the user gets an
      * uninitialized field" warning if they missed any.
      */
     bool useDataSegment;
 
-    FeatureSet features{};
+    FeatureSet features;
 
-    std::set<fs::path> containerConfigPaths{};
-    std::set<std::string> defaultHeaders{};
-    std::set<std::string> defaultNamespaces{};
-    std::vector<std::pair<std::string, std::string>> membersToStub{};
+    std::set<fs::path> containerConfigPaths;
```

Comment:
The `{}` makes no difference on these members, as class-objects are always default initialised if not initialised explicitly.

However, the error messages are a lot clearer without the `{}` and removing them allowed me to track down all the locations OICodeGen was being copied.